src/cs50.h

```diff
@@ -120,6 +120,11 @@ typedef char *string;
 #undef WARN
 #define WARN(func) func
 
+// If compiler isn't GCC or Clang it may not support __extension__, better to
+// suppress this error and hope for the best
```

Comment:
What's your heuristic for using `//` instead of `/* */`? (We tend not to capitalize if just a one-liner.)